From b31250e7e6e6e104674dc304ba74965bb56074d6 Mon Sep 17 00:00:00 2001
From: WANG Chao <chaowang@redhat.com>
Date: Thu, 3 Apr 2014 15:49:26 +0800
Subject: [PATCH] dracut-initqueue service runs before remote-fs-pre.target

With the following commit, dracut doesn't mount anything from /etc/fstab

commit e920bfb
Author: WANG Chao <chaowang@redhat.com>
Date:   Tue Apr 1 15:20:49 2014 +0800

    fstab: do not mount and fsck from fstab if using systemd

But systemd doesn't mount nfs at all, because no unit is pulling in
remote-fs.target.

dracut must pull in these remote fs mount and all these remote mounts
should start only after network is up (ie. after dracut-initqueue).

Signed-off-by: WANG Chao <chaowang@redhat.com>
---
 modules.d/98systemd/dracut-initqueue.service | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/98systemd/dracut-initqueue.service b/modules.d/98systemd/dracut-initqueue.service
index 03dcc4ebf5..1b9e701bea 100644
--- a/modules.d/98systemd/dracut-initqueue.service
+++ b/modules.d/98systemd/dracut-initqueue.service
@@ -11,6 +11,8 @@
 Description=dracut initqueue hook
 Documentation=man:dracut-initqueue.service(8)
 DefaultDependencies=no
+Before=remote-fs-pre.target
+Wants=remote-fs-pre.target
 After=systemd-udev-trigger.service
 Wants=systemd-udev-trigger.service
 ConditionPathExists=/etc/initrd-release
-- 
1.9.3

